test/test-meshctl-env.js

```diff
@@ -40,10 +40,11 @@ test('Test env commands', function(t) {
       exec.resetHome();
       exec(port, 'env-set 1 A=1 B=2', function(err, stdout) {
         tt.ifError(err, 'command should not error');
-        tt.equal(stdout, 'Service "service 1" environment updated\n' +
-          '    Name  Value\n' +
-          '    A     1\n' +
-          '    B     2\n',
+        tt.match(stdout,
+          new RegExp(['Service.+service 1.+environment updated\\n',
```

Comment:
@Setogit I don't understand this change.

After globalization, are there no longer quotes around `"service 1"`?